Units/parser-javascript.r/js-empty-class-name.d/expected.tags

```diff
@@ -1,2 +1,2 @@
-prop	input.js	/^var {prop} = { prop: "value" };$/;"	p	class:AnonymousClass4ca5b60a0101
-AnonymousClass4ca5b60a0101	input.js	/^var {prop} = { prop: "value" };$/;"	c
+prop	input.js	/^var {prop} = { prop: "value" };$/;"	p	class:anonymousObject4ca5b60a0105
```

Comment:
The value for scope field is `class:anonymousObject4ca5b60a0105`.
However, `anonymousObject4ca5b60a0105` is tagged as a variable.
So I think the value of the field should be:
```
variable:anonymousObject4ca5b60a0105
```

Reply:
Good point.  This requires that I tackle a FIXME in the code.  The patch is going to get bigger :)